CMakeLists.txt

```diff
@@ -17,42 +17,50 @@
 ###############################################################################
 
 cmake_minimum_required(VERSION 3.15)
-cmake_policy(SET CMP0091 NEW)
-
-set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded")
-set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /NODEFAULTLIB:MSVCRT /NODEFAULTLIB:MSVCRTD /NODEFAULTLIB:LIBCMTD /DEBUG:FULL /FORCE:MULTIPLE /IGNORE:4006,4075,4088,4099")
-
-set(CMAKE_MODULE_PATH "${CMAKE_BINARY_DIR}/vendor;${CMAKE_SOURCE_DIR}/cmake")
-
-option(SUPERBUILD "Build the project using a superbuild" ON)
-
-if (SUPERBUILD)
-	project(Makou_Reactor_SUPERBUILD LANGUAGES CXX)
-	set_directory_properties(PROPERTIES EP_BASE "${CMAKE_BINARY_DIR}/ep")
-	add_subdirectory(third_party)
-	include(ExternalProject)
-	ExternalProject_Add(
-		Makou_Reactor
-		SOURCE_DIR	"${CMAKE_SOURCE_DIR}"
-		CMAKE_ARGS
-			"-DSUPERBUILD=OFF"
-			"-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
-			"-DCMAKE_PREFIX_PATH=${CMAKE_BINARY_DIR}/vendor"
-			"-DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}"
-			"-DCMAKE_MSVC_RUNTIME_LIBRARY=${CMAKE_MSVC_RUNTIME_LIBRARY}"
-		DEPENDS
-		zlib
-	)
-	ExternalProject_Add_Step(
-		Makou_Reactor
-		reconfigure
-		COMMAND ${CMAKE_COMMAND} -E echo "Forcing a superbuild reconfigure"
-		DEPENDEES download
-		DEPENDERS configure
-		ALWAYS ON
-	)
-	return()
-endif()
+cmake_policy(SET CMP0074 NEW)
 
 project(Makou_Reactor LANGUAGES CXX)
-add_subdirectory(src)
+
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC ON)
+
+if(CMAKE_VERSION VERSION_LESS "3.7.0")
+    set(CMAKE_INCLUDE_CURRENT_DIR ON)
+endif()
+
+set(RELEASE_NAME "Makou_Reactor")
+set(RELEASE_VERSION "1.8.2")
+set(GUI_TARGET "${RELEASE_NAME}")
+
+find_package(Qt5 COMPONENTS Widgets OpenGL LinguistTools REQUIRED)
+find_package(ff7tk REQUIRED NO_MODULE COMPONENTS ff7tk ff7tkUtils)
+find_package(ZLIB REQUIRED)
+
+file(GLOB_RECURSE PROJECT_SOURCES "*.cpp" "*.h")
+
+# Translations
+file(GLOB_RECURSE translation_files "${RELEASE_NAME}_*.ts")
+set_source_files_properties(${translation_files} PROPERTIES OUTPUT_LOCATION "${CMAKE_INSTALL_FULL_BINDIR}")
+qt5_add_translation(QM_FILES ${translation_files})
+
+if(WIN32)
+    set (FF7TK_DEPENDS
+        ${ff7tk_DIR}/../../../bin/ff7tk.dll
+        ${ff7tk_DIR}/../../../bin/ff7tkWidgets.dll
```

Comment:
are you using widgets or utils?

Reply:
Utils, this line shouldn't be here yet, as FF7TK_DEPENDS is used for install rules.